### PR TITLE
vote: zhangyangyu as infra team member

### PR DESCRIPTION
**Pull request by Mini256**
As title.

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]
LGTM.

**Review by BusyJay — APPROVED** [review]

**Comment by Mini256** [comment]
@tisonkun Could you mind helping me merge this PR?

File: votes/0169-zhangyangyu-as-infra-team-member.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The vote will be open for at least 72 hours unless there is an objection or not enough votes.
+
+## Result
+
+Approved by 0 binding votes and 1 non-binding votes.
+
+* Mini256 (non-binding)
+* tisonkun (non-binding)
+* BusyJay (binding)
```
Comment (tisonkun) [inline votes/0169-zhangyangyu-as-infra-team-member.md:22]:
```suggestion
Approved by 1 binding votes and 2 non-binding votes.

* Mini256 (non-binding)
* tisonkun (non-binding)
* BusyJay (binding)
```

**Review by tisonkun — COMMENTED** [review]
> Consensus: Consensus requires 2 binding +1 votes and no binding vetoes.

Maybe @sunxiaoguang can give a review?

File: votes/0169-zhangyangyu-as-infra-team-member.md (hunk 1 above)
Comment (Mini256) [reply votes/0169-zhangyangyu-as-infra-team-member.md:22]: Good catch!

**Review by sunxiaoguang — APPROVED** [review]

**Review by sunxiaoguang — APPROVED** [review]
LGTM
+1 binding

**Pull request merged**